.github/workflows/assigned.yaml

```
@@ -0,0 +1,17 @@
+name: Issue Assigned
+
+on:
+  issues:
+    types: [assigned]
+
+  pull_request:
```

one small thing:
```suggestion
  pull_request_target:
```
this brings in the change to this file made to the version the certbot repo in https://github.com/certbot/certbot/pull/10224

otherwise, this lgtm!

---

ooh great catch thanks!